addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -41,13 +41,21 @@ def _get_l10n_in_base_tags(self):
            + self.env.ref("l10n_in.tax_tag_non_gst_supplies").ids
```

Comment:
@jco-odoo Well for the best we could just remove it from here?

Reply:
@hamo-odoo Just be careful to see if this method would not be used elsewhere.